CI/functional_tests/test_water_study.py

```diff
@@ -79,13 +79,14 @@ def mdsuite_project(traj_files, tmp_path) -> mdsuite.Project:
         traj_file_path=traj_files[2], topol_file_path=traj_files[0]
     )
 
-    project.add_experiment(
+    exp = project.add_experiment(
         name="water_sim",
         timestep=0.002,
         temperature=300.0,
         units=gmx_units,
         simulation_data=file_reader,
     )
+    exp.sample_rate = 5000
```

Comment:
Not for this PR, but we could consider having an `add_experiment(..., **kwargs)` and then do `[setattr(self, key, value) for key, value in kwargs.items]` so we can pass missing information directly through this method and don't have to call `exp.sample_rate` ?